pps_mw_training/pipelines/cloud_base/training_data.py

```diff
@@ -0,0 +1,142 @@
+from pathlib import Path
+from typing import Any
+import json
+import numpy as np  # type: ignore
+import tensorflow as tf  # type: ignore
+import xarray as xr  # type: ignore
+from pps_mw_training.utils.scaler import get_scaler
+
+
+def _load_data(
+    data_files: np.ndarray,
+    input_parameters: str,
+    label_parameters: str,
+    fill_value_input: float,
+    fill_value_label: float,
+) -> list[np.ndarray]:
+    """Load, scale, and filter data."""
+    with xr.open_mfdataset(
+        [f.decode("utf-8") for f in data_files],
+        combine="nested",
+        concat_dim="nscene",
+    ) as all_data:
+        input_params = json.loads(input_parameters)
```

Comment:
You are now doing the same operations for the input and label data, so it looks like we should have a function for this.
It would be nice if you define a function something like: `def scale_data(data: xr.Dataset, params, fill_value: float) -> np.ndarray:` and use it here.

Reply:
Yes, that is a good point. Now fixed